### DISCUSSION TITLE
Obsidian Looking Callouts

### DISCUSSION THREAD

**Discussion by Luke-zhang-04** [oleeskild/obsidian-digital-garden#151] [Ideas]
So I managed to make the callouts look like the ones in Obsidian.

![Screen Shot 2023-01-09 at 21 41 04](https://user-images.githubusercontent.com/55749227/211449628-20e258af-c47f-4cdf-a627-d281b857b1b7.png)

I also managed to make them collapsible, which would solve #89.

One problem is that is uses some pretty sketchy Javascript. The other problem is that this would change the appearance of existing callouts. I want to make a PR but not sure how to go about it. Should I make an option to enable these kinds of callouts, or just make them default?

**Comment by oleeskild** [comment]
Great! It would be very nice if you could make a PR. I think it's better that it looks closer to what users expect, which is how it looks on obsidian.

Don't worry about the wonky javascript. Make a PR, and I'll tweak it if it needs some changes.

**Reply by Luke-zhang-04** [reply]
Alright, it's https://github.com/oleeskild/digitalgarden/pull/43
